.github/workflows/update-nightlies.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           directory: ./third_party/contour-latest
           files: net-contour.yaml contour.yaml
```

Comment:
`hack/release.sh` generates those files as well as `release.yaml` in the `net-*` repos.
Why have both? Relying on a single file name `release.yaml` seems less brittle as it remove the need to know which the details of the components YAML.